.ng-dev/commit-message.ts

```diff
@@ -16,7 +16,5 @@ export const commitMessage: CommitMessageConfig = {
   maxLineLength: Infinity,
   minBodyLength: 0,
   minBodyLengthTypeExcludes: ['docs'],
-  scopes: [
-    ...Object.keys(packages).filter(name => !packages[name].private),
-  ],
+  scopes: Object.keys(packages),
```

Comment:
Looks good but can you leave the array in place.  It will cause less code churn if/when new scopes are added.

Reply:
Re added